pkg/exec/builder/action.go

```diff
@@ -73,20 +75,22 @@ func unmarshalActionMap(actionMap map[string][]interface{}, builder BuildableAct
 //		 err := yaml.Unmarshal(contents, &action)
 //		 return &action, err
 //	 })
-func LoadAction(cxt *portercontext.Context, commandFile string, unmarshal func([]byte) (interface{}, error)) error {
-	contents, err := readInputFromStdinOrFile(cxt, commandFile)
+func LoadAction(ctx context.Context, porterCtx *portercontext.Context, commandFile string, unmarshal func([]byte) (interface{}, error)) error {
+	//lint:ignore SA4006 ignore unused ctx for now
```

Comment:
What's the reason why we aren't just ignoring the return value using `_, span`

Reply:
Let's say that we did that to make lint happy. Then later someone needs to use the context variable. If they just use ctx, they will be using the context for the wrong scope, since we are throwing away the properly scoped context value on this line. They would also need to realize that they need to change that line. I don't want to make it easy to introduce a bug like that just to make lint happy. I'd rather set developers up to be successful and use a comment to make lint be quiet about it. 😁